eeg_hed_small/dataset_description.json

```diff
@@ -1,7 +1,7 @@
 {
     "Name": "Face processing MEEG dataset with HED annotation",
-    "BIDSVersion": "v1.4.0",
-    "HEDVersion": "8.0.0-alpha.2",
+    "BIDSVersion": "v1.8.2",
```

Comment:
The field `BIDSVersion` is not for the bids-validator version, it's for the version of the BIDS specification, which is currently 1.6 --> https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#dataset_descriptionjson

the bids-validator version and bids-specification version do not correspond

Reply:
Thanks for spotting that.  I've changed to v1.6.0